shared-bindings/gifio/OnDiskGif.c

```diff
@@ -199,6 +206,20 @@ MP_DEFINE_CONST_FUN_OBJ_1(gifio_ondiskgif_get_bitmap_obj, gifio_ondiskgif_obj_ge
 MP_PROPERTY_GETTER(gifio_ondiskgif_bitmap_obj,
     (mp_obj_t)&gifio_ondiskgif_get_bitmap_obj);
 
+//|     palette: displayio.Palette
```

Comment:
Is this right?

```suggestion
//|     palette: Optional[displayio.Palette]
```

Reply:
That covers that it can return `None`? if so yes that is right and I can commit that suggestion. It will always return something - even if it is None.

Reply:
Yup, it does.